cypress/defs.bzl

```diff
@@ -12,6 +12,30 @@ _cypress_test = rule(
     toolchains = js_binary_lib.toolchains + ["@aspect_rules_cypress//cypress:toolchain_type"],
 )
 
+def _cypress_test_macro(name, entry_point, cypress, **kwargs):
+    _cypress_test(
+        name = name,
+        entry_point = entry_point,
+        data = kwargs.pop("data", []) + [
+            cypress,
+        ],
+        chdir = native.package_name(),
+        enable_runfiles = select({
+            "@aspect_rules_js//js/private:enable_runfiles": True,
+            "//conditions:default": False,
+        }),
+        unresolved_symlinks_enabled = select({
+            "@aspect_rules_js//js/private:experimental_allow_unresolved_symlinks": True,
+            "//conditions:default": False,
+        }),
+        # Turn off sandboxing by default to allow electron to perform write
+        # operations. Cypress does not expose the underlying electron apis so we
+        # cannot alter the user app data directory to be within the bazel
+        # sandbox.
+        tags = kwargs.pop("tags", []) + ["no-sandbox"],
```

Comment:
Add an escape valve for users to enable the sandbox if they want to? Maybe they are on an older version of cypress where it works fine.

Sometimes I'll make the escape valve an undocumented attribute such as,
```
tags = kwargs.pop("tags", [])
if kwargs.pop("allow_sandbox", False):
  tags.append("no-sandbox")
```